### DISCUSSION TITLE
Use language codes without associated country

### DISCUSSION THREAD

**Discussion by itsnotsaved** [directus/directus#10637] [Feature Requests]
I noticed that currently directus store available languages with country code. e.g. af-ZA Please store them with language code by default and use country code only for especially required or significant contribution receive languages.

https://github.com/directus/directus/blob/main/app/src/lang/available-languages.yaml

When projects follow this practice, then people living in other countries or lands (or as minor groups in different regions) that use same language try to create separate versions for them only for political purposes. (or propose versions only has some synonyms difference or second versions contain many borrowed words from other languages)
Basically this harm to original languages with time and otherhand affect to the localization speed around world due split into groups

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
